sdv/lite/single_table.py

```diff
@@ -62,18 +62,6 @@ def __init__(self, metadata, name, locales=['en_US']):
         if name == FAST_ML_PRESET:
             self._setup_fast_preset(metadata, self.locales)
 
-    def add_constraints(self, constraints):
```

Comment:
should we just raise a deprecation warning here instead? Idk if we should delete any public functions